app/services/cookie_handler.rb

```diff
@@ -0,0 +1,42 @@
+class CookieHandler
+
+  AUTH_URL = 'http://authentication.andela.com/'
+  DATA_PATH = 'loggedin'
+  COOKIE_KEY = 'andela:session'
```

Comment:
I just have a feeling that seeing lines 3 - 5 in the codebase is too much information. I know it shouldn't be hacked but it can also go into `application.yml`.